app/admin/article.rb

```diff
@@ -0,0 +1,17 @@
+ActiveAdmin.register Article do
+
```

Comment:
Remember not to leave commented code. You can always get it back from git if you need it in the future